feature-account-impl/src/main/java/io/novafoundation/nova/feature_account_impl/domain/manualBackup/ManualBackupInteractor.kt

```diff
@@ -8,20 +8,55 @@ import io.novafoundation.nova.feature_account_api.domain.model.LightMetaAccount.
 import io.novafoundation.nova.feature_account_api.domain.model.LightMetaAccount.Type.POLKADOT_VAULT
 import io.novafoundation.nova.feature_account_api.domain.model.LightMetaAccount.Type.PROXIED
 import io.novafoundation.nova.feature_account_api.domain.model.MetaAccount
+import io.novafoundation.nova.feature_account_api.domain.model.hasAccountIn
+import io.novafoundation.nova.runtime.ext.defaultComparatorFrom
+import io.novafoundation.nova.runtime.multiNetwork.ChainRegistry
+import io.novafoundation.nova.runtime.multiNetwork.chain.model.Chain
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.map
+
+class MetaAccountChains(
+    val metaAccount: MetaAccount,
+    val defaultChains: List<Chain>,
+    val customChains: List<Chain>
+)
 
 interface ManualBackupInteractor {
```

Comment:
Not sure we need to put everything into single interactor. I suggest to create one interactor per screen